stl/inc/format

```diff
@@ -1006,18 +1008,19 @@ template <class _Context, class _Ty>
 
 template <class _Context, class _Ty>
 inline constexpr size_t _Get_format_arg_storage_size = sizeof(
-    _Get_format_arg_storage_type<_Context>(_STD declval<_Ty>()));
+    _Get_format_arg_storage_type<_Context>(static_cast<const _Ty&>(_STD declval<_Ty>())));

```

Comment:
So if I understand correctly, the issue is basically that a users construction of an rvalue `void*` `basic_format_arg` would result in a storage of a `_Pointer_type` but being handled through `make_format_args` makes a handle? Is that really an issue? The way that `basic_format_arg` is currently written, this compiles fine: `basic_format_arg<format_context>(static_cast<void*>(nullptr))`.

Reply:
My counterexample was a type with a user-defined conversion to `float` but only from non-`const` `meow`. I now believe it to be bogus since there's no way for a user to directly construct a `basic_format_args` - only the library can do so, and it does so only from functions that take `const Args&...` so they've already been `const`-ed when they get to this overload set.

TLDR: this comment is now simply:
```suggestion
    _Get_format_arg_storage_type<_Context>(_STD declval<const _Ty&>()));
```